bcipy/language/model/gpt2.py

```diff
@@ -81,84 +182,122 @@ def __get_char_predictions(self, word_prefix: str) -> List[tuple]:
 
         return char_prob_tuples
 
-    def __build_vocab(self) -> Dict[int, str]:
+    def __interpolate_language_models(self, lm1: Dict[str, float], lm2: Dict[str, float], coeff: float) -> List[Tuple]:
```

Comment:
This seems like it could be a utility function as well.

Reply:
Also changed it to a static method.